src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -535,15 +557,99 @@ def _visit_neighbors(self, node: gtir.FunCall) -> DataExpr:
             memlet=dace.Memlet(data=neighbors_temp, subset=neighbor_idx),
         )
 
-        assert isinstance(node.type, itir_ts.ListType)
         return DataExpr(neighbors_node, node.type)
 
+    def _visit_map(self, node: gtir.FunCall) -> DataExpr:
+        assert isinstance(node.type, itir_ts.ListType)
+        assert isinstance(node.type.element_type, ts.ScalarType)
+        dtype = dace_utils.as_dace_type(node.type.element_type)
+
+        input_args = [self.visit(arg) for arg in node.args]
+
+        # TODO(edopao): extract offset_dim from the input arguments
+        offset_dim = gtx_common.Dimension("", gtx_common.DimensionKind.LOCAL)
+        map_index = dace_gtir_utils.get_map_variable(offset_dim)
+        connectors = [f"__arg{i}" for i in range(len(input_args))]
+
+        # The dataflow we build in this class has some loose connections on input edges.
+        # These edges are described as set of nodes, that will have to be connected to
+        # external data source nodes passing through the map entry node of the field map.
```

Comment:
I still do not think that views are the best solution here.
Take a look at `SDFGState.add_memlet_path()` that can creates paths through multiple maps.

But I still does not see how the view helps, as you have to connect that thing still with the outside.